CHANGELOG.md

```diff
@@ -1,3 +1,119 @@
+# 0.33.0 (2020-02-??)
+
+## Language changes
+
+### Macros
+
+- Add `TypeNode.name(generic_args : BoolLiteral)` to return `TypeNode`'s name with or without type vars. ([#8483](https://github.com/crystal-lang/crystal/pull/8483), thanks @Blacksmoke16)
+
+## Standard library
+
+- **(breaking-change)** Remove several deprecated methods and modules. ([#8646](https://github.com/crystal-lang/crystal/pull/8646), [#8596](https://github.com/crystal-lang/crystal/pull/8596), thanks @bcardiff, @Blacksmoke16)
```

Comment:
Technically, these features have already been named in the respective change notes when they were tagged as deprecated. But in case the deprecation was unnoticed / not reacted to, this release is where code using these features will fail. So maybe it's better to at least name them explicitly here. It helps grepping the changelog to find information about the method that's breaking the code. ``
`PartialComparable`, `Crypto::Bcrypt::Password#==`, `HTTP::Server::Response#respond_with_error`, `JSON::PullParser::Kind#==`, `Symbol#==(JSON::PullParser::Kind)`, `JSON::Token#type`, `String#at`, `Time.new`, `Time.now`, `Time.utc_now`, `URI.escape`, `URI.unescape` ``

Reply:
May want to phrase it as "several previously deprecated methods" just...for people like me to not get confused? :)